utils/getUpdated.js

```diff
@@ -0,0 +1,14 @@
+// Get last updated time from corona.lmao.ninja
+const axios = require("axios");
+const chalk = require("chalk");
+const logSymbols = require("log-symbols");
+const cyan = chalk.cyan;
+
+module.exports = async () => {
+	const api = await axios.get(`https://corona.lmao.ninja/all`);
+	let data = Object.values(api.data);
+
+	console.log(
+		`${logSymbols.info} ${cyan(`Last Updated:`)} ${Date(data[3])}`
```

Comment:
Shouldn't this be `new Date()`

Reply:
It shouldn't matter since I'm not storing or manipulating it. The API just returns an integer containing the UNIX timestamp of the last updated time, so I just use the Date() function to make it human readable.